### ISSUE TITLE
[Conformance][networking][router] openshift router metrics

### ISSUE THREAD

**Issue by sosiouxme** [openshift/origin#14871]
[Test failure](https://ci.openshift.redhat.com/jenkins/job/merge_pull_request_openshift_ansible/629/consoleFull#-12956338556cbb9a5e4b02b88ae8c2f77) that seems unlikely to be related to the [PR](https://github.com/openshift/openshift-ansible/pull/4495).

```
[Conformance][networking][router] openshift router metrics
/go/src/github.com/openshift/origin/test/extended/router/metrics.go:172
  The HAProxy router
  /go/src/github.com/openshift/origin/test/extended/router/metrics.go:171
    should expose a health check on the metrics port [It]
    /go/src/github.com/openshift/origin/test/extended/router/metrics.go:64

    Expected error:
        <*errors.errorString | 0xc4203fdb20>: {
            s: "host command failed: error running &{/data/src/github.com/openshift/origin/_output/local/bin/linux/amd64/kubectl [kubectl --server=https://ip-172-18-12-225.ec2.internal:8443 --kubeconfig=/etc/origin/master/admin.kubeconfig exec --namespace=extended-test-router-metrics-m03wv-9t217 execpod -- /bin/sh -c curl -s -o /dev/null -w '%{http_code}' \"http://172.18.12.225:1935/healthz\"] []  <nil> 000  [] <nil> 0xc420c730e0 exit status 7 <nil> <nil> true [0xc4200316f0 0xc420031708 0xc420031720] [0xc4200316f0 0xc420031708 0xc420031720] [0xc420031700 0xc420031718] [0xcae5e0 0xcae5e0] 0xc42144d020 <nil>}:\nCommand stdout:\n000\nstderr:\n\nerror:\nexit status 7\n\n",
        }
        host command failed: error running &{/data/src/github.com/openshift/origin/_output/local/bin/linux/amd64/kubectl [kubectl --server=https://ip-172-18-12-225.ec2.internal:8443 --kubeconfig=/etc/origin/master/admin.kubeconfig exec --namespace=extended-test-router-metrics-m03wv-9t217 execpod -- /bin/sh -c curl -s -o /dev/null -w '%{http_code}' "http://172.18.12.225:1935/healthz"] []  <nil> 000  [] <nil> 0xc420c730e0 exit status 7 <nil> <nil> true [0xc4200316f0 0xc420031708 0xc420031720] [0xc4200316f0 0xc420031708 0xc420031720] [0xc420031700 0xc420031718] [0xcae5e0 0xcae5e0] 0xc42144d020 <nil>}:
        Command stdout:
        000
        stderr:
        
        error:
        exit status 7
        
        
    not to have occurred

    /go/src/github.com/openshift/origin/test/extended/router/metrics.go:63
------------------------------
[BeforeEach] [Top Level]
  /go/src/github.com/openshift/origin/test/extended/util/test.go:51
```

**Comment by sosiouxme** [comment]
More detailed [log.txt](https://github.com/openshift/origin/files/1099317/log.txt)

**Comment by sdodson** [comment]
Related to https://github.com/openshift/origin/pull/14790 ?

**Comment by smarterclayton** [comment]
Yeah, that's the wrong port.  Will fix the test tomorrow, but not sure why it's running in the other job but not on the merge queue

**Comment by sdodson** [comment]
> Yeah, that's the wrong port. Will fix the test tomorrow, but not sure why it's running in the other job but not on the merge queue

@stevekuznetsov any idea why this problem wasn't found via the origin merge jobs but has blocked openshift-ansible test/merge?

**Comment by stevekuznetsov** [comment]
Looks like it's failing on a pure install, not an install + update to latest. Origin queue does not run both, just the update.

**Comment by smarterclayton** [comment]
OK, going to fix it (the fact that we aren't running the test in the queue
is because we don't set this on upgrade for various reasons).

**Comment by smarterclayton** [comment]
Fixed in #14889
